src/NHibernate.Test/Async/SecondLevelCacheTest/InvalidationTests.cs

```diff
@@ -14,7 +14,6 @@
 using System.Reflection;
 using NHibernate.Cache;
 using NHibernate.Cfg;
-using NHibernate.Engine;
```

Comment:
Missing async regen for a small change done on a previously merged PR. Unrelated but so small, let it get fixed here.